### ISSUE TITLE
Coralium rarity is a roadblock to progression.

### ISSUE THREAD

**Issue by BryanBoru** [Darkosto/SevTech-Ages#685]
Beta3

- it's frequency of use in the later advancements of Stage 1
- No land samples to help identify a location makes it that one or two ores absent from the geolosys system we become accustomed to while playing STA
- Biomes it generates in are also based on RNG of seed and location
- only one or two blocks in the group when you do happen to find it
- It's an abrupt choke point and hurdle to progression for getting past Stage 1. Not based on skill or preparation, but solely RNG.

_I understand exploration is a goal of the pack. I hope to not come off as complaint, but honestly this is frustrating beyond gameplay preferences of step-up or missing mod preferences and the like. I have spent the past ~~two~~ three days with zero results, having branch mining any Coralium Swamp I can find as my only option left. Even finding the swamps are a difficult. My guess is most players will get frustrated about this situation._

**Comment by BryanBoru** [comment]
Understood. 
I had been concentrating on swamps, Ocean depths are significant and getting under them was intimidating.
The infected swamps led to fails but for the two blocks I found. 

I only found out about the worldgen banding last night and it helped me focus my searching to stop moving longitudinal and staying latitudinal in fitting wet/warm bands. 
Halfway through the morning today I switched methods and built a ladder tower down to seafloor and did eventually find some, very low Y. In fact, in both cases there were more than two blocks, which was a nice find.

In review of all of this, including @Darkosto's reply, I feel as though just a bit of clarity, or information given to the player may help avoid the frustration I was facing.
In my case, I feel discussing it with other testers and using information I obtained watching the devstream helped me.
The future player may not have those avenues to get information. Thanks for the reply.
